ocaml/xapi/xapi_globs.ml

```diff
@@ -558,22 +558,29 @@ let event_hook_auth_on_xapi_initialize_succeeded = ref false
 (** Xenclient enabled *)
 let xenclient_enabled = false
 
+(* bios_string length is limited to 512 characters *)
+let bios_string_limit_size = 512
+
 (** {2 BIOS strings} *)
 
 (** Type 11 strings that are always included *)
 let standard_type11_strings =
   ["oem-1", "Xen";
    "oem-2", "MS_VM_CERT/SHA1/bdbeb6e0a816d43fa6d3fe8aaef04c2bad9d3e3d"]
 
-(** Generic BIOS strings *)
-let generic_bios_strings =
+(** Standard BIOS strings - settable by user *)
+let standard_bios_strings =
```

Comment:
I think "standard" and "generic" are not helpful names.
Maybe "customisable" for the ones the user can specify, and "all" or "default" for the entire list of customisable_bios_strings + const_bios_strings?